src/libslic3r/GCode/GCodeProcessor.cpp

```diff
@@ -60,6 +60,7 @@ const std::vector<std::string> GCodeProcessor::Reserved_Tags = {
     "_GP_LAST_LINE_M73_PLACEHOLDER",
     "_GP_ESTIMATED_PRINTING_TIME_PLACEHOLDER",
     "_GP_TOTAL_LAYER_NUMBER_PLACEHOLDER",
+    " CHANGE_TOOL ",
```

Comment:
It's not clear to me of this tag.
We didn't write this tag anywhere, so the new added codes will never be excuted.
Could you explain the purpose of this?

Reply:
I was using this tag as a replacement for the standard T0 etc. The tag is used when the gcode is being processed. Without using something similar to this, the gcode viewer would not display the tool/color change. Is this not what tags are used for?

Reply:
@Ocraftyone I'm not clear why we introduce a new tag here
Tool change is identified and processed by parsing tool change command(refer to process_gcode_line function), depends on the firmware type, it could be T or M108/M135 command. It's dedicated command hence we don't a tag for that.

Reply:
Yeah. So if you just completely remove any command (say remove the custom tag and the T or M108/M135) The gcode preview will not show the color change on the model (it will all stay the original color). That tag is there so that the gcode preview can read the tool change but it stays commented out so that the printer will not process the T command (because for manual swaps, we don't want the printer to do anything except the custom gcode). 

With Tag Enabled:
![image](https://github.com/SoftFever/OrcaSlicer/assets/24759591/00348730-f518-4c3e-b87e-3ceb997dee7e)

Without Tag Enabled:
![image](https://github.com/SoftFever/OrcaSlicer/assets/24759591/da98f175-01cb-4b93-8470-0d03e0a79747)

Reply:
Ah, I see.
Can we rename it to "MANUAL_TOOL_CHANGE" to better describe the use?